geometry_msgs/msg/PolygonInstanceStamped.msg

```diff
@@ -0,0 +1,6 @@
+# This represents a Polygon with reference coordinate frame and timestamp
+# Includes and unique identification field for disambiguating multiple instances
+
+std_msgs/Header header
+geometry_msgs/Polygon polygon
+int64 id
```

Comment:
Sorry, one more quibble, this would make it more in-line with the other `FooStamped` messages.

```suggestion
std_msgs/Header header
geometry_msgs/PolygonInstance polygon
```